### PR TITLE
Fixes collisions with libextobjc

### PR DESCRIPTION
**Pull request by Air-Craft**
I use libextobjc and there were issues with the fact that you've changed the "namespace" on some of the functions but not on the names of the files.  My includes were getting the libextobjc files and showing compilation errors for your namespaced functions.  I've fixed this so it builds properly.  You might want to go through and namespace everything, including the macro's to ensure there's no versioning issues.  Hope this helps...

### CODE REVIEW

**Comment by Club15CC** [comment]
Sorry I missed this. If you still care...

The issues is that you have 

#import "EXTScope.h"

...But you've made modifications from the original `libextobjc` version of that file (i.e. you've added the functions prefixed with `mtl_`).  I had `EXTScope.h` already in my project and when Mantle does the `#import` it's getting my version (the original) which is missing those mtl functions thus causing the error. 

Basically this pull request just changes the EXTScope.h filename to a Mantle-specific name to coincide with the fact that you've made Mantle-specific modifications.

**Comment by jspahrsummers** [comment]
@Club15CC That would only cause problems if you're importing Mantle files one-by-one. If you import the Xcode project, build the framework, and import headers that way, you shouldn't run into filename collisions.

**Comment by Club15CC** [comment]
That may be true, and forgive me for being slightly pedantic, but if you use a file from another project, modify it and add your own namespaced functions, why would you not namespace the file too? Or make a separate file for the additions and include the original one as-is?

**Comment by jspahrsummers** [comment]
Files don't need to be namespaced because imports should already be scoped correctly. If they're not, then there's a configuration error.
